runtime/lib/component/light.js

```diff
@@ -18,6 +18,8 @@ function Light (runtime) {
   EventEmitter.call(this)
   this.runtime = runtime
   this.dbusRegistry = runtime.component.dbusRegistry
+
+  this.ttsSoundCnt = {}
```

Comment:
What does this `cnt` mean?

Reply:
named ttsSoundTask ?

Reply:
`cnt` stands for Count. I'd change the naming.

Reply:
Yea, I think count would be more readable, cnt and cut are confusion a bit :)